<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_registro_pasajeros</class>
 <widget class="QDialog" name="ventana_registro_pasajeros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de pasajeros</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	border-color: rgb(0,0,0)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>102</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_pasajero">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>920</width>
         <height>310</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>920</width>
         <height>310</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
	border-width: 1px;
	border-style: solid;
	border-color: rgb(131, 131, 131);
	border-radius: 5px;
}
QLabel{
	font: bold 15px;
	background-color: transparent;
}

QLabel#lbl_mensaje2{
	color:rgb(255, 0, 0);
	font: 5pt &quot;Calibri&quot;;

}

QLineEdit{
	background-color: rgb(255,255,255);
	font: 12pt &quot;Calibri&quot;;
	border-width: 2px;
	border-top-color: rgb(0, 0, 0);
	border-left-color: rgb(0, 0, 0);
	border-right-color: rgb(255,255,255);	
	border-bottom-color: rgb(255,255,255);
	border-style: solid;

}

QPushButton{
	background-color: rgb(255,255,255);
	font: 12pt &quot;Calibri&quot;;
	border-width: 2px;
	border-style: solid;
	border-color: rgb(131, 131, 131);
	border-radius: 4px;
}

QPushButton:hover{
	background-color: rgb(153, 142, 140);
	font: 75 italic 12pt &quot;Calibri&quot;;
	border-width: 2px;
	border-style: solid;
	border-color: rgb(131, 131, 131);
	border-radius: 4px;
}</string>
       </property>
       <property name="title">
        <string>Carga de datos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>351</x>
          <y>62</y>
          <width>72</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>NOMBRE:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_nombre</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nombre">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>60</y>
          <width>391</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>62</y>
          <width>104</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>DOCUMENTO:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_dni</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dni">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>137</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>122</y>
          <width>94</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>LOCALIDAD:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_localidad</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_localidad">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>241</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>431</x>
          <y>122</y>
          <width>94</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>PROVINCIA:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_provincia</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_provincia">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>120</y>
          <width>281</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>173</y>
          <width>73</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>PATENTE:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_patente</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_patente">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>170</y>
          <width>137</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>221</y>
          <width>53</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>EMAIL:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_email</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>371</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>171</y>
          <width>84</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>VEHICULO:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_vehiculo</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_vehiculo">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>170</y>
          <width>211</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>501</x>
          <y>222</y>
          <width>86</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>TELEFONO:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_telefono</cstring>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_telefono">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>220</y>
          <width>137</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cancelar">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>270</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>CANCELAR</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_siguiente">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>270</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>SIGUIENTE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_finalizar">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>270</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>FINALIZAR</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_mensaje">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>*Campo Obligatorio</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_mensaje_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>90</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>*Campo Obligatorio</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>112</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="tabla_mostrar_pasajeros">
     <property name="minimumSize">
      <size>
       <width>1149</width>
       <height>370</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1149</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>141</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>141</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>DOCUMENTO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NOMBRE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LOCALIDAD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PROVINCIA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VEHICULO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PATENTE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EMAIL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TELEFONO</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_dni</tabstop>
  <tabstop>lineEdit_nombre</tabstop>
  <tabstop>lineEdit_localidad</tabstop>
  <tabstop>lineEdit_provincia</tabstop>
  <tabstop>lineEdit_patente</tabstop>
  <tabstop>lineEdit_vehiculo</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_telefono</tabstop>
  <tabstop>btn_cancelar</tabstop>
  <tabstop>btn_siguiente</tabstop>
  <tabstop>btn_finalizar</tabstop>
  <tabstop>tabla_mostrar_pasajeros</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
